### ISSUE TITLE
Add support to select by UUID

### ISSUE THREAD

**Issue by jsvini** [ponyorm/pony#116]
When I try to do a select like this:

```python

class Client(db.Entity):
  id = PrimaryKey(UUID, default=uuid4)

Client.select().filter(id=clientId)
```
I get this error:
```python
NotImplementedError: <class 'uuid.UUID'>
```

**Comment by smokingkapor** [comment]
any update for this issue @amalashkevich

**Comment by kozlovsky** [comment]
Thanks @jsvini for the reporting and @smokingkapor for the reminding!
This bug should be fixed now. The fix will be part of the release PonyORM 0.6.2

**Comment by smokingkapor** [comment]
thank you :-) @kozlovsky
